src/osuTK.iOS/osuTK.iOS.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="MSBuild.Sdk.Extras/1.6.55">
+  <Import Project="..\Common.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+    <PackageVersion>1.0.8</PackageVersion>
+    <PackageId>ppy.osuTK.iOS</PackageId>
+    <Title>ppy.osuTK.iOS</Title>
+    <RootNamespace>osuTK.iOS</RootNamespace>
+    <AssemblyName>osuTK.iOS</AssemblyName>
+    <IPhoneResourcePrefix>Resources</IPhoneResourcePrefix>
+    <TargetFramework>xamarinios10</TargetFramework>
+    <OutputPath>bin/$(Configuration)/</OutputPath>
```

Comment:
The reason for this is because of the project import that I mentioned. Can you check again with removing both of these lines? I can build it just fine.